components/kyma-environment-broker/internal/model.go

```diff
@@ -78,9 +78,12 @@ type ProvisioningOperation struct {
 	AVSEvaluationExternalId int64 `json:"avs_evaluation_external_id"`
 }
 
-// DeprovisioningOperation holds all information about provisioning operation
+// DeprovisioningOperation holds all information about de-provisioning operation
 type DeprovisioningOperation struct {
 	Operation `json:"-"`
+
+	AVSInternalEvaluationDeleted bool `json:"avs_internal_evaluation_deleted"`
```

Comment:
I'd create separate structs for AVSFlags and AVSIDs under one struct AVS which I'd use here. reason: look comment below